system_test/common/aest_db_SUITE.erl

```diff
@@ -151,8 +162,14 @@ sc_leave_upgrade_reestablish( {{BeforeNodeNameI, BeforeNodeTypeI}, {BeforeNodeNa
     HostPathR = node_db_host_path(BeforeNodeNameR, Cfg),
 
     OldVersion = proplists:get_value(state_channels_vsn, Cfg),
-    OldNodeF = fun(NodeName, DbHostPath) -> node_spec(NodeName, OldVersion, DbHostPath, true) end,
-    NewNodeF = fun(NodeName, DbHostPath) -> node_spec(NodeName, "local", DbHostPath, true) end,
+    OldNodeF =
+        fun(NodeName, DbHostPath) ->
+            node_spec(NodeName, OldVersion, DbHostPath, is_mining_node(NodeName))
```

Comment:
This solves the issue with a couple of nodes mining concurently, thus producing forks